libraries/objectDefaultFiles/object.js

```diff
@@ -101,6 +100,29 @@
     // this will be initialized once the frame creates a new SpatialInterface()
     var realityInterface = null;
 
+    let urlObj = {};
+
+    function getIoTitle (title) {
```

Comment:
This seems to be used a lot, but I can't quite decipher what it's doing (what is urlObj.n and when is it defined or not?) – do you mind adding a comment for future readability?

Reply:
Added some comments, thanks for pointing this out!